mobile/src/main/java/ch/epfl/sdp/mobile/infrastructure/persistence/store/Store.kt

```diff
@@ -8,4 +8,16 @@ interface Store {
 
   /** Accesses a collection with the given [String] identifier. */
   fun collection(path: String): CollectionReference
+
+  /**
+   * Runs the give [Transaction] on the [Store], executing all the operations atomically. A
+   * [Transaction] must perform all its reads before performing all of its writes.
+   *
+   * This function may throw an [Exception] if a transaction can't be executed.
```

Comment:
Can we add this information with a keyword, something like `@Exception` ? I don't if it's possible

Reply:
Fixed in d1b4b2e0ff48edc3a1438212e25d1472214c6741.